src/core/include/cesium/omniverse/GltfUtil.h

```diff
@@ -73,6 +75,11 @@ struct FeaturesInfo {
     std::vector<FeatureId> featureIds;
 };
 
+struct ImageryLayersInfo {
+    uint64_t imageryLayerCount;
+    std::vector<OverlayType> overlayTypes;
+};
```

Comment:
I got rid of imageryLayerCount. I've kept the struct because now we have two vectors in it. One is for the type (ion vs polygon) and one is for how the rendering is treated (overlay vs clipping)